parse.go

```diff
@@ -64,9 +64,20 @@ type command struct {
 	dest        path
 	specs       []*spec
 	subcommands []*command
+	groups      []*command
 	parent      *command
 }
 
+// getSpecs returns all specs of the command, plus all the option groups.
+func (cmd command) getSpecs() []*spec {
```

Comment:
just call this `specs`

Reply:
That did conflict with the `specs` struct attribute, but since most cases would call this function it made sense to rework the code to accommodate.